crates/context/src/lib.rs

```diff
@@ -748,6 +758,28 @@ impl ContextManager {
         Ok(ids)
     }
 
+    pub fn get_own_signing_key(
```

Comment:
rename this method, we need to be able to fetch "sender" (not signing) keys for any member in the context